scripts/launcher/lib/log_monitor/log_handler.py

```diff
@@ -42,6 +44,16 @@ class VirtualLogRequestHandler(LogRequestHandler):
         "Call Trace:"
     ]
 
+    # Path to the config file
+    CONFIG_FILE_PATH = '/tmp/ignored_simple_tests.conf'
+
+    # Read additional ignored messages from the config file
+    if os.path.isfile(CONFIG_FILE_PATH):
+        with open(CONFIG_FILE_PATH, 'r') as config_file:
+            extra_ignored_tests = [line.strip() for line in config_file if line.strip()]
+            # Extend the ignored_simple_tests list
+            ignored_simple_tests.extend(extra_ignored_tests)
```

Comment:
This solution is not inline with what we are doing elsewhere and also it will be taken by all the tests in parallel because the file is available to all.

Reply:
You're right, it didn't occur to me that it actually behaves that way. I've edited the code and it should be fine now. I've tested everything.